SignalServiceKit/src/Util/OWSAsserts.h

```diff
@@ -139,6 +139,18 @@ NS_ASSUME_NONNULL_BEGIN
     } while (NO)
 
 
+#define OWSProdExit(_messageFormat, ...)                                                                               \
+    do {                                                                                                               \
+        OWSFail(_messageFormat, ##__VA_ARGS__);                                                                        \
+        exit(0);                                                                                                       \
+    } while (0)
+
+#define OWSCProdExit(_messageFormat, ...)                                                                              \
+    do {                                                                                                               \
+        OWSCFail(_messageFormat, ##__VA_ARGS__);                                                                       \
+        exit(0);                                                                                                       \
+    } while (NO)
```

Comment:
Options:

* We could add these new `OWSExit()` macros and then migrate (most?) `OWSFail()` macros to use them.  
* Same, but we could call them `OWSProdFail()` and rename the existing analytics macros with those names to something else.
* We could simply call the exiting macros `OWSFail()` and rename the existing macros to `OWSFailWithoutExit()` or whatever.  WDYT?

Reply:
Done, with `OWSFailDebug()`.